src/pages/generate/pages/loading/ProgressBar.tsx

```
@@ -114,6 +115,8 @@ const ProgressLoadingBar = ({ onComplete }: ProgressLoadingBarProps) => {
           style={{
             animation: 'slideUp 0.4s ease-out',
             margin: 0,
+            whiteSpace: 'nowrap',
```

ESLint

[error] Replace `'nowrap'` with `"nowrap"`

(prettier/prettier)